recipes/elfio/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conans import ConanFile, tools
+
+
+class ElfioConan(ConanFile):
+    name = "elfio"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://elfio.sourceforge.net"
+    description = "A header-only C++ library that provides a simple interface for reading and generating files in ELF binary format."
+    topics = ("conan", "elfio", "elf")
+    license = "MIT"
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("elfio-{}".format(self.version), self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package(self):
+        self.copy("COPYING", src=os.path.join(self.source_folder, self._source_subfolder), dst="licenses")
+        self.copy(pattern=os.path.join("elfio", "*.hpp"), src=os.path.join(self.source_folder, self._source_subfolder), dst="include")
```

Comment:
```suggestion
        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
        self.copy(pattern=os.path.join("elfio", "*.hpp"), src=self._source_subfolder, dst="include")
```

Reply:
what is the difference ?

Reply:
`self.source_folder` is not required

Reply:
just shorter and relative to the current location? 
Ok